PlayBook/src/getting_started/download_playcover.md

```diff
@@ -2,22 +2,18 @@
 
 ## Prerequisites
 
-At the moment, PlayCover can only run on Apple Silicon Macs. Devices with the following chips are supported:
-
-* M1
-* M1 Pro
-* M1 Max
-* M1 Ultra
-* M2
+At the moment, PlayCover only works on **Apple Silicon Macs**. The minimum hardware requirement is an Apple M1 chip, PlayCover will also work on all Apple Silicon chips released after the M1.
 
 If you have an Intel Mac, you can explore alternatives like Bootcamp or emulators.
 
 ## Download
 
-You can download stable releases [here](https://github.com/PlayCover/PlayCover/releases), or build from source by following the instructions below.
+You can download the [latest stable release](https://github.com/PlayCover/PlayCover/releases/latest), or build from source by following the instructions below. 
+
+Additionally, you can download the [latest nightly version](https://nightly.link/PlayCover/PlayCover/workflows/2.nightly_release/develop/PlayCover_nightly_210.dmg) to test out experimental features not yet added to the stable release.
```

Comment:
https://nightly.link/playcover/playcover/workflows/2.nightly_release/develop is better instead.